### PR TITLE
MONGOID-5330 reverse feature flag for 7.5

### PR DESCRIPTION
**Pull request by Neilshweky**

### CODE REVIEW

**Review by p-mongo — APPROVED** [review]
Reverse for 8.0?

**Comment by Neilshweky** [comment]
> Reverse for 8.0?

This is actually reversing the feature flag for 7.5 as you can see the base of the PR is 7.5-stable. The value as true for overwrite_chained_operators is the current functionality in Mongoid.

**Comment by p-mongo** [comment]
:+1:

File: docs/reference/configuration.txt [hunk 1]
```diff
@@ -395,8 +395,8 @@ for details on driver options.
       # When chaining the same operators that use the same field, setting this
       # feature flag to false will cause those operators to be combined using an
       # and. Setting this feature flag to true will cause the later chained
-      # operators to overwrite the earlier ones. (default: false)
-      #overwrite_chained_operators: false
+      # operators to overwrite the earlier ones. (default: true)
```
Comment (johnnyshields) [inline docs/reference/configuration.txt:398]: Please follow the comment format used elsewhere in this file, if this is indeed the case:

```
      # (default: true in Mongoid 7, will change to false in Mongoid 8)
```

**Review by comandeo — APPROVED** [review]

**Pull request merged**